src/components/Members.tsx

```diff
@@ -3,17 +3,21 @@ import styles from './Members.module.css';
 
 import MemberCard from "./MemberCard";
 
-type Member = {
+export type Member = {
+  // should implement a few custom types later
   readonly name: string;
-  readonly image: string;
-  readonly website: string;
-  readonly github: string;
-  readonly linkedin: string;
-  readonly bio: string;
-  readonly favoredLink: string;
+  readonly bio?: string; // should be shorter than 120 characters
+  readonly image?: string;
+  readonly website?: string;
+  readonly github?: string;
+  readonly linkedin?: string;
+  readonly email?: string; // should begin with "mailto:"
+  readonly favoredLink?: "website" | "github" | "linkedin" | "email";
 };
```

Comment:
I'm thinking about making these types more granular 

For example:
- `email` must begin with `mailto:` followed by the email address
- `bio` should be a string no longer than 120 characters
- `linkedin` and `github` should begin with their domain